projects/plugins/jetpack/modules/woocommerce-analytics/classes/class-jetpack-woocommerce-analytics-trait.php

```diff
@@ -262,7 +262,7 @@ public function get_common_properties() {
 			'ui'                                 => $this->get_user_id(),
 			'url'                                => home_url(),
 			'woo_version'                        => WC()->version,
-			'store_admin'                        => in_array( 'administrator', wp_get_current_user()->roles, true ) ? 1 : 0,
+			'store_admin'                        => in_array( 'administrator', wp_get_current_user()->roles, true ) || in_array( 'shop_manager', wp_get_current_user()->roles, true ) ? 1 : 0,
```

Comment:
Hey @jimjasson - What do you think of having one `in_array` instead of two?

```php
in_array( array( 'administrator', 'shop_manager' ), wp_get_current_user()->roles, true )
```